OCPBUGS-60065# Add RN for automatic CNS volume deletion. (#97050)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

Previously, the self-signed loopback certificate for the Kubernetes API Server expired after one year. With this release, the expiration date of the certificate is extended to three years.

> [discrete]

Reviewer: ```suggestion

```

Author: De-discreted all headings in this section.

[id="ocp-release-notes-readiness-probes-etcd_{context}"]
=== Readiness probes exclude etcd checks

The readiness probes for the API server have been modified to exclude etcd checks. This prevents client connections from being closed if etcd is temporarily unavailable. This means that client connections persist through brief etcd unavailability and minimizes temporary API server outages.

> [discrete]

Reviewer: ```suggestion

```

[id="ocp-4-19-notable-technical-changes-remove-cns_{context}"]
=== Installer automatically removes leftover Cloud Native Storage (CNS) volumes

> The OpenShift installer now automatically detects and removes leftover persistent storage volumes on {vmw-full} when you delete a cluster. This prevents orphaned volumes from consuming disk space and creating unnecessary alerts in vCenter.

Reviewer: ```suggestion
The {product-title} installation program now automatically detects and removes leftover persistent storage volumes on {vmw-full} when you delete a cluster. This prevents orphaned volumes from consuming disk space and creating unnecessary alerts in vCenter.
```

Author: Added suggested change.
